.codeclimate.yml

```diff
@@ -0,0 +1,2 @@
+exclude_paths:
+- app/models/snapshot.rb
```

Comment:
Is there a reason to exclude this file? The commit message for this change could explain that.